GameFiles/_integration.gsc

```diff
@@ -35,6 +36,20 @@ init()
     level.eventCallbacks[level.eventTypes.clientDataReceived]       = ::OnClientDataReceived;
     level.eventCallbacks[level.eventTypes.executeCommandRequested]  = ::OnExecuteCommand; 
     level.eventCallbacks[level.eventTypes.setClientDataCompleted]   = ::OnSetClientDataCompleted;
+
+    level.clientCommandCallbacks = [];
+    level.clientCommandRusAsTarget = [];
+    //Populate collections above
```

Comment:
unneeded comment imo

Reply:
I would keep it to make it clear that they are not empty and what the intended way of populating them is.
Of course that can quickly be looked up in the code, its your call if we keep it or not, I just thought I make it clearer for other people that might want to add new commands.